### PR TITLE
4439-Update List-item-groups

### PR DESCRIPTION
**Pull request by Liliya-web**

### CODE REVIEW

File: components/groups/list_item_groups/NoActionListItemGroup.vue [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<template>
+  <v-card
+    class="mx-auto"
+    max-width="400"
+  >
+    <v-list flat>
```
Comment (omega790) [inline components/groups/list_item_groups/NoActionListItemGroup.vue:6]: It should be
```
<v-list-group> 
  ... 
   <v-list-group> 
      ...
   </v-list-group>
</v-list-group>
```

For example:
```suggestion
<template>
  <v-card class="mx-auto" max-width="400">
    <v-list>
      <v-list-group :value="true" no-action>
        <template v-slot:activator>
          <v-list-item-title>Test</v-list-item-title>
        </template>
        <v-list-group :value="true" color="indigo" no-action sub-group>
          <template v-slot:activator>
            <v-list-item-content>Test2</v-list-item-content>
          </template>
          <v-list-item v-for="(item, i) in items" :key="i" link>
            <v-list-item-icon>
              <v-icon v-text="item.icon"></v-icon>
            </v-list-item-icon>
            <v-list-item-content>
              <v-list-item-title v-text="item.text"></v-list-item-title>
            </v-list-item-content>
          </v-list-item>
        </v-list-group>
      </v-list-group>
    </v-list>
  </v-card>
</template>```

Comment (Liliya-web) [reply components/groups/list_item_groups/NoActionListItemGroup.vue:6]: Done. But there is an issue. If the list-item-group is closed once, it's not opening again

Comment (omega790) [reply components/groups/list_item_groups/NoActionListItemGroup.vue:6]: I suppose you have a error in implementation
I've checked with my example and it works correct

**Review by omega790 — COMMENTED** [review]
Please fix a bug

File: components/groups/list_item_groups/NoActionListItemGroup.vue (hunk 1 above)
Comment (Liliya-web) [reply components/groups/list_item_groups/NoActionListItemGroup.vue]: Fixed

**Review by omega790 — APPROVED** [review]

**Pull request merged**